config/scripts/add-header-webcomponent.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# To be properl executed by maelstro initialization need the +x flag 
```

Comment:
# This script needs to be executable in order to run during the initialization phase
 # It might be required to `chmod +x` the file